src/scss/components/_footer.scss

```diff
@@ -0,0 +1,191 @@
+
+        .ufb-notice--notice-row--3tUOv {
+            .txt {
+                padding: $padding--24--0;
+            }
+        }
+
+        .image-operate {
+            .img-operate {
+                margin-right: $margin-right--24;
+                margin-bottom: $margin-bottom--24;
+                vertical-align: middle;
+
+                &:last-child {
+                    margin-right: 0;
+                }
+            }
+        }
+    }
+
+    .footer-section-main {
+        background: $color-white;
+        border-top: $border-size-default solid $color-border-gray;
+        padding: $padding--0--32;
+        padding-top: $footer-section-main-padding-top;
+
+        .links-and-locale {
+
+            .locale-select-container {
+                padding: 0;
+                order: 1;
+                flex-shrink: 0;
+                margin-left: auto;
+            }
+
+            .unstyled-list {
+                list-style: none;
+                margin: 0;
+                padding: 0;
+                max-width: none;
+            }
+
+            .link-column {
+                flex-basis: 256px;
+                margin-right: $margin-right--16;
+                margin-bottom: 0;
+
+                li {
+                    padding-left: 0;
+
+                    .link {
+                        font-weight: $font-weight-medium;
+                        line-height: $line-height--1-4;
+                        display: block;
+                    }
+                }
+            }
+        }
+
+        .locale-select-container {
+            padding: 0;
+            order: 1;
+            flex-shrink: 0;
+            margin-left: auto;
+
+            .form-group {
+                max-width: $max-width-600;
+            }
+        }
+
+        .form-label {
+            display: flex;
+            align-items: center;
+            margin: 0;
+            padding-bottom: $padding-bottom--8;
+            min-height: $min-height--28;
+        }
+
+        .sr-only {
+            position: absolute;
+            height: $height--1;
+            width: $width--1;
+            overflow: hidden;
+            clip: $form-label-clip;
+        }
+
+        .select-container {
+            position: relative;
+            min-width: $min-width--240;
+            max-width: $max-width-600;
+
+            .select-icon-container {
+                pointer-events: none;
+                display: flex;
+                align-items: center;
+                position: absolute;
+                top: 0;
+                height: $height-full;
+                padding: $padding--0--8;
+
+
+                &.select-icon-left {
+                    left: 0;
+                    justify-content: flex-end;
+                    width: $width--36;
+                }
+
+                &.select-icon-right {
+                    right: 0;
+                    width: $width--36;
+                }
+            }
+        }
+
+        .select {
```

Comment:
Could you please create new components/_select.scss
And move this logic into that file